src/run.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const logger = require('winston');
+const parser = require('js-yaml');
+const fs = require('fs');
+const _ = require('lodash');
+
+let spawnSync = require('child_process').spawnSync;
+
+function expandTokens(command, vars) {
```

Comment:
@jbachhardie @AtebMT 

How do we feel here about functions? From what I've seen in OS land folk seem to prefer this style

```js
const expandTokens = () => {}
```